lib/teiserver/autohost/tachyon_handler.ex

```diff
@@ -105,11 +141,76 @@ defmodule Teiserver.Autohost.TachyonHandler do
     {:stop, :normal, 1000, [{:text, resp}], state}
   end
 
+  # Generic handler for when a response arrives too late
```

Comment:
I am planning to refactor the transport bits, so that this logic about awaiting for the response and timeout management will be centralised and abstracted out of the Autohost and Player modules.